common/models/agent-trace.json

```diff
@@ -5,15 +5,18 @@
     "processId": {
       "type": "number",
       "description": "id of ServiceProcess, not the pid",
-      "required": true
+      "required": true,
+      "postgresql": {"column": "processId"}
```

Comment:
I wish we had comments :-(

What does this mean? Why doesn't it default to the name of the field?